manifests/a/ApacheFriends/Xampp/8/1/8.1.10-0/ApacheFriends.Xampp.8.1.installer.yaml

```
@@ -15,7 +15,7 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerType: exe
-  InstallerUrl: https://downloadsapachefriends.global.ssl.fastly.net/8.1.10/xampp-windows-x64-8.1.10-0-VS16-installer.exe
+  InstallerUrl: https://netix.dl.sourceforge.net/project/xampp/XAMPP%20Windows/8.1.10/xampp-windows-x64-8.1.10-0-VS16-installer.exe

```

```suggestion
  InstallerUrl: https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/8.1.10/xampp-windows-x64-8.1.10-0-VS16-installer.exe/download
```
Because SourceForge selects the CDN which is closest to the user, it might be best to not specify which CDN server to use